modules/apps/users-admin/users-admin-impl/src/main/java/com/liferay/users/admin/internal/configuration/settings/UserFileUploadsSettingsImpl.java

```diff
@@ -26,31 +26,65 @@ public class UserFileUploadsSettingsImpl implements UserFileUploadsSettings {
 
 	@Override
 	public int getImageMaxHeight() {
-		return _userFileUploadsConfiguration.imageMaxHeight();
+		try {
+			UserFileUploadsConfiguration userFileUploadsConfiguration =
+				_configurationProvider.getCompanyConfiguration(
+					UserFileUploadsConfiguration.class,
+					CompanyThreadLocal.getCompanyId());
+
+			return userFileUploadsConfiguration.imageMaxHeight();
+		}
+		catch (ConfigurationException configurationException) {
+			return ReflectionUtil.throwException(configurationException);
+		}
 	}
 
 	@Override
 	public long getImageMaxSize() {
-		return _userFileUploadsConfiguration.imageMaxSize();
+		try {
+			UserFileUploadsConfiguration userFileUploadsConfiguration =
+				_configurationProvider.getCompanyConfiguration(
+					UserFileUploadsConfiguration.class,
+					CompanyThreadLocal.getCompanyId());
+
+			return userFileUploadsConfiguration.imageMaxSize();
+		}
+		catch (ConfigurationException configurationException) {
```

Comment:
Could you create a function for getting configuration in this class?

Reply:
It might be similar to this change https://github.com/liferay/liferay-portal/commit/e37ac2e01184662a78c2f2eb570a54afa50b75ff

Reply:
Fixed it. Thank you all.